services/management/backup/backups_logic_service.go

```diff
@@ -0,0 +1,348 @@
+// pmm-managed
+// Copyright (C) 2017 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+package backup
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"gopkg.in/reform.v1"
+
+	"github.com/percona/pmm-managed/models"
+)
+
+// BackupsLogicService represents core logic for backuping.
+type BackupsLogicService struct {
+	db          *reform.DB
+	jobsService jobsService
+	l           *logrus.Entry
+}
+
```

Comment:
I think it's better to move this service to separate directory `/services/backup/`

Reply:
Any name suggestions?
Right now we got services/backup and services/management/backup. so 2x same package name, not sure if we can improve it.

And also if  `BackupsLogicService`is great name :D

Reply:
As it will be located in the completely separate package it can be simple `Service`

Reply:
And for the packag name, I think it's not big deal to have the same name `backups`